File: Misc. Algos/Recursive_algos/factorial.c
// Finding factorial of a given number
// code by Akshat Aryan
#include <stdio.h>

int factorial(int n){
    if(n == 1){
        return 1;
    }
    return (n * factorial(n - 1));
}

int main(){
    int a;
    printf("Enter your number:");
    scanf("%d", &a);
    printf("Factorial = %d\n", factorial(a));
    return 0;
}

File: Misc. Algos/Recursive_algos/factorial.py
# Finding factorial of a given number
# code by Akshat Aryan

def factorial(n):
    if(n == 1):
        return 1
    return n * factorial(n - 1)

if(__name__ == "__main__"):
    a = int(input("Enter your number:"))
    print("Factorial =", factorial(a))